docs/tutorials/patch_update.md

```diff
@@ -0,0 +1,35 @@
+## Creating a Patch File
+
+We can also create a patch file for the update and changes that we made to the source code.
+
+1. Assuming we have made a change and we want to submit a patch.
+
+If necessary, update R-devel
+
+```bash
+cd $TOP_SRCDIR
+svn update
+cd $BUILDDIR
+make # this will run the configure as before
```

Comment:
I think we need to update this for the new $TOP_SRCDIR and $BUILDDIR. I suggest we always use "R-devel" as the default name for the built version

```suggestion
If necessary, update the version of R you are working on, with the latest changes in <https://svn.r-project.org/R/trunk/>. E.g., if the version of R you are working on is named "R-devel-working"

```bash
cd $TOP_SRCDIR/R-devel-working
svn update
cd $BUILDDIR/R-devel-working
make # this will run the configure as before
```